openedx/features/philu_courseware/views.py

```diff
@@ -4,80 +4,82 @@
 
 from django.utils.translation import ugettext as _
 from rest_framework import status
-from rest_framework.decorators import api_view, renderer_classes
-from rest_framework.renderers import BrowsableAPIRenderer, JSONRenderer
+from rest_framework.exceptions import APIException, ValidationError
 from rest_framework.response import Response
+from rest_framework.views import APIView
+from submissions.api import SubmissionError
 
-from lms.djangoapps.courseware.courseware_access_exception import CoursewareAccessException
-from lms.djangoapps.courseware.exceptions import CourseAccessRedirect
+from courseware.models import StudentModule
+from lms.djangoapps.instructor import enrollment
```

Comment:
How about just import them method `reset_student_attempts` instead of import the whole module?